docs/chain-specification-standard.md

```diff
@@ -24,31 +24,55 @@ to make sure that clients will be able to properly handle your definition.
       ]
     },
     "coins": {
-      "type": "array",
-      "items": [
-        {
-          "type": "object",
-          "properties": {
-            "name": {
-              "type": "string"
-            },
-            "symbol": {
-              "type": "string"
-            },
-            "denom": {
+      "type": "object",
+      "properties": {
+        "defaultTypes": {
+          "type": "array",
+          "items": [
+            {
               "type": "string"
-            },
-            "exponent": {
-              "type": "integer"
             }
-          },
-          "required": [
-            "name",
-            "symbol",
-            "denom",
-            "exponent"
+          ]
+        },
+        "list": {
+          "type": "array",
+          "items": [
+            {
+              "type": "object",
+              "properties": {
+                "name": {
+                  "type": "string"
+                },
+                "symbol": {
+                  "type": "string"
+                },
+                "denom": {
+                  "type": "string"
+                },
+                "types": {
```

Comment:
Can we find a more specific term instead of "types"? This word is so overloaded.

features? flags? properties?

Reply:
`properties` would be the best option to me. @pedrouid opinion?